### PR TITLE
Bug 1468092 - Convert lodash .sum() to native ES6

### PR DESCRIPTION
**Pull request by rajpatel507**

### CODE REVIEW

File: ui/js/services/perf/compare.js [hunk 1]
```diff
@@ -84,7 +84,10 @@ treeherder.factory('PhCompare', [
                     let stddev;
 
                     if (testName === 'Noise Metric') {
-                        average = Math.sqrt(_.sum(values.map(x => x ** 2)));
+                        average = Math.sqrt(values.map(x => x ** 2).filter((finalList, num) => {
```
Comment (edmorley) [inline ui/js/services/perf/compare.js]: I think `.reduce((a, b) => a + b)` might be shorter?

File: ui/js/services/perf/math.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+
```
Comment (edmorley) [inline ui/js/services/perf/math.js]: Could you also remove the newlines at the start of this file? :-)

Comment (edmorley) [reply ui/js/services/perf/math.js]: (In addition to the change here: https://github.com/mozilla/treeherder/pull/3675#discussion_r195943456)

**Comment by rajpatel507** [comment]
Updated PR as per comment.

File: ui/js/services/perf/compare.js [hunk 3]
```diff
@@ -84,7 +84,7 @@ treeherder.factory('PhCompare', [
                     let stddev;
 
                     if (testName === 'Noise Metric') {
-                        average = Math.sqrt(_.sum(values.map(x => x ** 2)));
+                        average = Math.sqrt(values.map(x => x ** 2).filter((a, b) => a + b, 0));
```
Comment (edmorley) [inline ui/js/services/perf/compare.js]: I think the `, 0` needs to be removed (leftover from when using `.reduce()`)

**Comment by rajpatel507** [comment]
it should be` .reduce()` instead of `.filter()`, corrected code now.

**Pull request merged**